exercises/practice/bob/.meta/config.json

```diff
@@ -1,6 +1,18 @@
 {
   "blurb": "Bob is a lackadaisical teenager. In conversation, his responses are very limited.",
   "authors": [],
```

Comment:
author not on github

https://github.com/exercism/csharp/tree/4d73a89429b3eda350c3107c4f36091e792284a6/bob